eth/abc.py

```diff
@@ -473,7 +587,7 @@ def add_transaction(self,
     def get_block_transactions(
             self,
             block_header: BlockHeaderAPI,
-            transaction_class: Type[SignedTransactionAPI]) -> Sequence[SignedTransactionAPI]:
+            transaction_class: Type[SignedTransactionAPI]) -> Tuple[SignedTransactionAPI, ...]:
```

Comment:
Another non-doc change sneaked in here. It seems more aligned with our best practices to have the return type be the actual type that is returned (in contrast to parameter types which should accept the broadest type that the method could work with). This also aligns the method with the other methods on that class that do also use `Tuple[Something, ...]` as return type.